### ISSUE TITLE
`canvg@3.0.8` cannot be required because it added `"type": "module"` to its package.json

### ISSUE THREAD

**Issue by trxcllnt** [fuzhenn/node-svg2img#59]
I left a comment in [this `canvg` issue](https://github.com/canvg/canvg/issues/1480#issuecomment-920352999), but it seems 3.0.8 added `"type": "module"` to their package.json, causing node v16 to treat `canvg/lib/index.js` as ESM and disallowing `require("canvg")`:

```
$  node --input-type=commonjs --eval "require('canvg')"
node:internal/modules/cjs/loader:1150
      throw err;
      ^

Error [ERR_REQUIRE_ESM]: require() of ES Module node_modules/canvg/lib/index.js from [eval] not supported.
index.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in node_modules/canvg/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).

    at [eval]:1:1
    at Script.runInThisContext (node:vm:129:12)
    at Object.runInThisContext (node:vm:305:38)
    at [eval]-wrapper:6:22 {
  code: 'ERR_REQUIRE_ESM'
}
```

Might be good to pin to `canvg@3.0.7` if this takes a while to resolve.

**Comment by iambumblehead** [comment]
Ive run into this issue also https://github.com/canvg/canvg/issues/1481

**Comment by tsabbay** [comment]
Same here, node 16.9 is throwing ERR_REQUIRE_ESM as an error

**Comment by iambumblehead** [comment]
this package worked well for me https://www.npmjs.com/package/svg-render and so sharing it with everyone here in this ticket.

**Comment by jubarbie** [comment]
Same for me, I can't use this module. Thanks @iambumblehead for sharing this other module

**Comment by kgn** [comment]
Thanks @iambumblehead, trying to switch to this to convert remote SVGs but can't seem to get this fed into the buffer property correctly, is this something you were able to get working? https://github.com/catdad-experiments/svg-render/issues/7

**Comment by fuzhenn** [comment]
https://github.com/fuzhenn/node-svg2img/commit/a05d4701825d0c7c8f9c55db49b4d9a69aab3acd
I sticked to canvg@3.0.7 which is the last version before canvg changed type to module, and released a fix version `0.9.4`.
It should be working for now and I will find some time to fix this.
